icons/insomnia/insomnia-original-wordmark.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
```

Comment:
Optimize, just copy the code from [this suggestion](https://github.com/devicons/devicon/pull/1513#discussion_r1036172614)